CHANGES.md

```diff
@@ -1,8 +1,9 @@
 # Release Notes
 
-## 0,5,2 (TDB)
+## 0.5.2 (TDB)
```

Comment:
I think this should be a minor version update because it's really breaking the WMTSCapatibilities endpoint (removed the openapi schemas for dependencies and return xml with multiple layers)